### PR TITLE
Refactor PUT (related to toggle button) to be restplus

### PR DESCRIPTION
**Pull request by MinervaMint**

### CODE REVIEW

**Comment by MinervaMint** [comment]
The application on cloud was not working, with "Booting worker with pid: xx" occurring infinitely. So I changed the memory assigned to each instance from 64M to 128M for now to make it work, but I am not sure why this could solve the problem. Maybe anyone has any thoughts?
------------ UPDATE --------------
So I guess it just simply ran out of memory.

File: service/service.py [hunk 1]
```diff
@@ -74,6 +87,25 @@
     "status", type=inputs.boolean, required=False, help="List Recommendations by status"
 )
 
+
+######################################################################
+# Authorization Decorator
```
Comment (sushanthsamala) [inline service/service.py]: I think we agreed on not having an API key right?

**Review by sushanthsamala — CHANGES_REQUESTED** [review]

File: service/service.py (hunk 1 above)
Comment (MinervaMint) [reply service/service.py:92]: Ah yes, I forgot to clean this part up. Thanks for reminding, I will delete this now.

Comment (MinervaMint) [reply service/service.py]: > I think we agreed on not having an API key right?

Ah yes, I forgot to clean this part up. Thanks for reminding, I will delete this now.

**Review by sushanthsamala — APPROVED** [review]

**Pull request merged**